module/importer-ergonode-1/src/Infrastructure/Processor/ErgonodeImportProcess.php

```diff
@@ -61,6 +62,7 @@ public function start(Import $import): void
             }
         } catch (Throwable $exception) {
```

Comment:
this case case is handled in Ergonode\Importer\Infrastructure\Handler\Import\StartImportCommandHandler
Your solution only change import error message. 
IMHO this is not a solution of this problem, and this logger is redundant.